src/nvim/regexp_nfa.c

```diff
@@ -2347,7 +2347,6 @@ static void nfa_set_code(int c)
 
 }
 
-#ifdef REGEXP_DEBUG
```

Comment:
This `#ifdef REGEXP_DEBUG` is entirely contained in a second `#ifdef REGEXP_DEBUG` so I removed it. This situation was created when the debug ifdefs in the regexp code were consolidated into just one, `REGEXP_DEBUG`.